tap_gorgias/client.py

```diff
@@ -8,7 +8,7 @@
 
 LOGGER = singer.get_logger()
 
-DEFAULT_TIMEOUT = os.getenv('DEFAULT_HTTP_TIMEOUT')
+DEFAULT_TIMEOUT = os.getenv('DEFAULT_HTTP_TIMEOUT') and int(os.getenv('DEFAULT_HTTP_TIMEOUT'))
```

Comment:
```suggestion
timeout = os.getenv('DEFAULT_HTTP_TIMEOUT')
try:
    DEFAULT_TIMEOUT = int(timeout) if timeout else None
except (TypeError, ValueError):
    DEFAULT_TIMEOUT = None
```

We can decide to set the `DEFAULT_TIMEOUT` on exception to be some value like 10 or None.